README.md

```diff
@@ -5,6 +5,13 @@ Consist of 2 parts:
 + Telegram bot to play quests.
 + Service for quest creating including back-end and front-end.
 
+To play quests user must use telegram bot. In bot player need to choose quest and then quest session starts.
+
+Quests are creating with web service. Creators make quests with sequence of stations.
+After creating quest become unpublic and only chosen users can play.
```

Comment:
I think "After creation,  the quest becomes private and only selected users can play it" sounds better